vm/src/import.rs

```diff
@@ -14,45 +14,57 @@ use crate::vm::{InitParameter, VirtualMachine};
 #[cfg(feature = "rustpython-compiler")]
 use rustpython_compiler::compile;
 
-pub fn init_importlib(vm: &mut VirtualMachine, initialize_parameter: InitParameter) -> PyResult {
+pub fn init_importlib(
```

Comment:
this is receiving `vm` as an argument but requries to call `enter_vm`. this is only called in initialization step, so it will not make any problem. but this is making a bit uncomfortable feeling.

Reply:
Should I make init_importlib `pub(crate)`?

Reply:
And yeah, that's definitely tricky

Reply:
i didn't think about `pub(crate)` at that time, but it looks like to be. Even if there is a need to manally init importlib, it must be a method of vm or interpreter rather than this kind of function. I think we'd better to list up pub apis. Sometimes internal stuffs are marked as pub just to be shared in the crate.